axelor-account/src/main/java/com/axelor/apps/account/service/move/massentry/MassEntryVerificationServiceImpl.java

```diff
@@ -456,7 +456,8 @@ public void verifyCompanyBankDetails(
       Move move, Company company, BankDetails companyBankDetails, Journal journal)
       throws AxelorException {
     BankDetails newCompanyBankDetails = companyBankDetails;
-    int technicalTypeSelect = journal.getJournalType().getTechnicalTypeSelect();
+    int technicalTypeSelect =
+        journal.getJournalType() != null ? journal.getJournalType().getTechnicalTypeSelect() : 0;
```

Comment:
Please create a global variable for journal type called "NO_TYPE" or something similar